lib/topological_inventory/mock_source/entity.rb

```diff
@@ -104,7 +104,7 @@ def resource_version_default_value
       end
 
       def resource_version_timestamp
-        Time.new.to_i
+        Time.new.to_s
```

Comment:
This generates "2019-05-16 12:42:40 +0200" instead of "1558003368"
Is it what you want, or `Time.new.to_i.to_s`?

Reply:
Both are good, unless you work with this somewhere as with integer

Reply:
The openapi schema defines this as string, so that is all that is needed for passing the ingress validation

Reply:
thinking generally about speed - timestamp is there for skip saving data with the same timestamp. 
String comparison is significantly slower than integer comparison, so what's the reason?

Even for this case, "2019-05-16 12:42:40 +0200" is slower to compare than "1558003368"

Reply:
Hm I think this perf difference will be lost since we are now comparing all attributes in the persister. (not just the resource_version like before)

Reply:
In this case (beyond scope of this PR) - is `resource_version` column needed?

Reply:
Btw please change it to `to_i.to_s`, because word `timestamp` means number

Reply:
done